sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Blobs/src/Config/BlobsExtensionConfigProvider.cs

```diff
@@ -205,6 +210,35 @@ public async Task<HttpResponseMessage> ConvertAsync(HttpRequestMessage input, Ca
             return (T)blob.BlobClient;
         }
 
+        private static ParameterBindingData ConvertToParameterBindingData(BlobAttribute blobAttribute)
+        {
+            var blobPath = BlobPath.ParseAndValidate(blobAttribute.BlobPath);
+
+            dynamic blobDetails = new ExpandoObject();
+            blobDetails.Connection = blobAttribute.Connection ?? Constants.DefaultAzureStorageConnectionName;
```

Comment:
If you look at the logic the other blob bindings use for resolving a connection https://github.com/Azure/azure-sdk-for-net/blob/cb3c61938d63c63313ae0fb8de876f7a42bdaf81/sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Blobs/src/Config/BlobsExtensionConfigProvider.cs#L429 there's handling for %% name resolver strings, as well as connection prefixing logic such that you can specify "Storage" or "AzureWebJobsStorage" and they'll resolve to the same thing. If our goal here is to resolve the connection in the same way for deferred, we'll need similar logic.

Reply:
I can switch to using the named resolver, but I'm not sure about the "Storage" connection string - the default we always tell customers to use is `AzureWebJobsStorage` so I felt it made most sense to default to that - does "Storage" ever get used?

Reply:
Yeah we can use "storage" as below -

```csharp
[Function("Function1")]
public void Run([BlobTrigger("testblob/{name}", Connection = "Storage")] string myBlob, string name)
{
      _logger.LogInformation($"C# Blob trigger function Processed blob\n Name: {name} \n Data: {myBlob}");
}
```

`local.settings.json` -
```json
{
  "Values": {
    "Storage": "<connection string>",
    "FUNCTIONS_WORKER_RUNTIME": "dotnet-isolated"
  }
}
```